library/core/src/slice/mod.rs

```diff
@@ -4540,6 +4540,120 @@ impl<T> [T] {
         // are disjunct and in bounds.
         unsafe { Ok(self.get_many_unchecked_mut(indices)) }
     }
+
+    /// Finds the index that an element points to within the slice.
```

Comment:
"element points to" doesn't sound right since the elements aren't necessarily pointers. Maybe

```suggestion
    /// Locate a pointed-to item within a slice from its reference.
```

Reply:
Good catch. I'm going to go with `Returns the index that an element reference points to` for now.